README.md

```diff
@@ -1 +1,14 @@
-# react-hw
\ No newline at end of file
+1. Create new repository for react homeworks with an appropriate name
```

Comment:
Readme file shouldn't include task description :)
It should contain information about your project set up